html/components/_masthead.scss

```diff
@@ -348,12 +348,27 @@
   padding: $sprk-masthead-accordion-item-padding;
   text-decoration: none;
 
+  // Button User Agent Resets
+  min-width: 100%;
```

Comment:
min-width: 100%? this isnt something thats used too much, whats it for in this case?

Reply:
user agent buttons have min-width 0% which makes the button not take up its full container.

Reply:
changed this to width 100% instead per our convo!